### PR TITLE
Itay/hello world vue

### PR DESCRIPTION
**Pull request by itaycohen**

### CODE REVIEW

File: apps/hello-world-vuejs/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    "css-loader": "^0.28.7",
+    "file-loader": "^1.1.4",
+    "node-sass": "^4.5.3",
+    "sass-loader": "^6.0.6",
+    "vue-loader": "^13.0.5",
+    "vue-template-compiler": "^2.4.4",
+    "webpack": "^3.6.0",
+    "webpack-dev-server": "^2.9.1"
+  },
+  "license": "MIT",
```
Comment (benyitzhaki) [inline apps/hello-world-vuejs/package.json]: probably not critical, but what license should we use?

File: apps/hello-world-vuejs/src/App.vue [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      settings: "",
+      context: "",
+      user: ""
+    };
+  },
+
+  created() {
+    monday.listen("settings", this.getSettings);
+    monday.listen("context", this.getContext);
+    this.fetchUserName();
```
Comment (benyitzhaki) [inline apps/hello-world-vuejs/src/App.vue]: i think that it should be:
```
monday.api(`query { me { name } }`).then(this.getUser);
```

and add a method bellow that will be identical to the "getSettings"

File: apps/hello-world-vuejs/src/App.vue [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  },
+
+  methods: {
+    beforeCreate() {
```
Comment (orrgottlieb) [inline apps/hello-world-vuejs/src/App.vue]: i'm not familiar with vue but is beforeCreate like componentwillmount or the constructor i think it should be on the mounted life cycle

File: apps/hello-world-vuejs/package.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  "name": "hello-world-vue",
```
Comment (orrgottlieb) [inline apps/hello-world-vuejs/package.json]: maybe we should change the names to something like monday.com-hello-world-app-vue

File: apps/hello-world-vuejs/index.html [hunk 5]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>hello-world-vue</title>
+  </head>
+  <body>
+    <div id="app"></div>
+    <script src="/dist/build.js"></script>
```
Comment (orrgottlieb) [inline apps/hello-world-vuejs/index.html]: type="text/javascript"

File: apps/hello-world-vuejs/index.html [hunk 6]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>hello-world-vue</title>
```
Comment (orrgottlieb) [inline apps/hello-world-vuejs/index.html]: same name as in the package.json

File: apps/hello-world-vuejs/package.json [hunk 7]
```diff
@@ -0,0 +1,33 @@
+{
+  "_from": "hello-world-vue",
+  "bundleDependencies": false,
+  "dependencies": {
+    "vue": "^2.5.11",
+    "monday-sdk-js": "^0.0.9"
+  },
+  "deprecated": false,
+  "description": "A Vue.js project",
```
Comment (orrgottlieb) [inline apps/hello-world-vuejs/package.json]: need to add some description imo

**Pull request merged**